sdk/src/main/java/io/dapr/client/DaprClientGrpc.java

```diff
@@ -66,6 +71,39 @@
     this.stateSerializer = stateSerializer;
   }
 
+  private CommonProtos.StateOptions.StateConsistency getGrpcStateConsistency(StateOptions options) {
+    switch (options.getConsistency()) {
+      case EVENTUAL:
+        return CommonProtos.StateOptions.StateConsistency.CONSISTENCY_EVENTUAL;
+      case STRONG:
+        return CommonProtos.StateOptions.StateConsistency.CONSISTENCY_STRONG;
+      default:
+        throw new IllegalArgumentException("Unsupported consistency model");
+    }
+  }
+
+  private CommonProtos.StateOptions.StateConcurrency getGrpcStateConcurrency(StateOptions options) {
+    switch (options.getConcurrency()) {
+      case FIRST_WRITE:
+        return CommonProtos.StateOptions.StateConcurrency.CONCURRENCY_FIRST_WRITE;
+      case LAST_WRITE:
+        return CommonProtos.StateOptions.StateConcurrency.CONCURRENCY_LAST_WRITE;
+      default:
+        throw new IllegalArgumentException("Unsupported consistency model");
```

Comment:
Todo: better Exception.

Reply:
IllegalArgmentException is a well known runtime exception. What is your idea here?

Reply:
Sorry, not exception type, exception message.